build/periodic.sh

```diff
@@ -22,11 +22,11 @@ cloneRepos() {
 		REPOS=$(cat policy-grc-squad/main-branch-sync/repo.txt)
 		# Manually append deprecated repos
 		REPOS="${REPOS}
-			stolostron/grc-ui
-			stolostron/grc-ui-api
 			stolostron/governance-policy-spec-sync
 			stolostron/governance-policy-status-sync
-			stolostron/governance-policy-template-sync"
+			stolostron/governance-policy-template-sync
+			stolostron/policy-collection
+			stolostron/policy-generator-plugin"
```

Comment:
Should we also add `governance-policy-framework-addon`?

Reply:
No. It's already in `repo.txt`: https://github.com/stolostron/policy-grc-squad/blob/35f8a8646b751ce8a8039b938a4e3dbee3aad051/main-branch-sync/repo.txt#L6

Reply:
These are the repos that either don't have an image build or have dropped from our current version.